client/dashboard/utils/site-slug.ts

```diff
@@ -0,0 +1,9 @@
+import { urlToSlug, withoutHttp } from 'calypso/lib/url/http-utils';
```

Comment:
In other places we just call `new URL().hostname`, wouldn't that be enough? In what cases does this fail?

Reply:
I think you're right. Updated to use that so that we don't need to import from calypso/lib/url!

Reply:
Nice!